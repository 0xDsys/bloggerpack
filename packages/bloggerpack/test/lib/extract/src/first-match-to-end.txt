<extract-start>
foo
</nope>

<extract-start>
bar
</nope>

<extract-start>
baz
</extract-end>
