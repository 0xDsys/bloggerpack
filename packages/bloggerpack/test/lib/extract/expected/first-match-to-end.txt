/* src/first-match-to-end.txt */

foo
</nope>

<extract-start>
bar
</nope>

<extract-start>
baz
